lib/Zureg/SendEmail.hs

```diff
@@ -27,8 +29,20 @@ data Handle = Handle
 
 withHandle :: Config -> (Handle -> IO a) -> IO a
 withHandle hConfig f = do
-    hAwsEnv <- Aws.newEnv Aws.Discover
+
+    -- AWS region is not retrieved correctly from environment variables, and neither from the AWS profile.
+    awsRegion <- regionFromEnv
+    hAwsEnv <- Aws.newEnv Aws.Discover <&> maybe id (Aws.envRegion .~) awsRegion
```

Comment:
Thanks for the PR, this all looks good!  Do you think we can move the duplication to a separate module?  I would propose either:

- `Zureg.Aws.smartRegion`
- `Network.AWS.Extended.smartRegion`

Reply:
Moved it to `Network.AWS.Extended`!